src/main/cli/cli.c

```diff
@@ -1422,10 +1420,7 @@ static void cliAdjustmentRange(char *cmdline)
             ptr = nextArg(ptr);
             if (ptr) {
                 val = atoi(ptr);
-                if (val >= 0 && val < MAX_SIMULTANEOUS_ADJUSTMENT_COUNT) {
-                    ar->adjustmentIndex = val;
-                    validArgumentCount++;
-                }
+                validArgumentCount++;
```

Comment:
Really minor - but maybe add a comment as to what this argument position was and that it's been removed but the command wasn't changed due to legacy reasons.

Reply:
@etracer65: Good point, added a comment.